![Logo TUBAF](https://tu-freiberg.de/sites/default/files/media/freiberger-alumni-netzwerk-6127/wbm_orig_rgb_0.jpg)

Exercise material of the BSc-level course **Mechanical Properties of Loose Rocks and Soils**.
Held at Technische Universität Bergakademie Freiberg.

Comments to:

*Prof. Dr. Thomas Nagel  
Chair of Soil Mechanics and Foundation Engineering  
Geotechnical Institute  
Technische Universität Bergakademie Freiberg.*

https://tu-freiberg.de/en/fakultaet3/gt/soilmechanics

# Lagerungsdichte und Materialbedarf bei Tiefenverdichtung

Hinweis: Das Beispiel wurde aus Fellin (2018) "Geotechnik - Lernen mit Beispielen" entnommen.

## Problemstellung und Eingangsgrößen

Eine locker gelagerte Sandschicht der Mächtigkeit $z_\text{max}$ wurde mit Tiefenrüttlern auf einer Fläche von $a \times b$ verdichtet. Dabei wurde ein quadratisches Raster mit dem Rasterabstand $r$ verwendet. Das Grundwasser steht in $z_\text{GW}$ Tiefe unter GOK an.

In [1]:
z_max = 10 #m
z_GW = 2 #m
a = 30 #m
b = 50 #m
r = 2 #m

Die folgenden Kennwerte des Sandes vor der Verdichtung seien bekannt: Kornwichte $\gamma_\text{s}$, Wichte im erdfeuchten Zustand $\gamma$ beim Wassergehalt $w$ (über GW), sowie die Porenzahl bei lockerster und dichtester Lagerung $n_\text{max}$ und $n_\text{min}$.

In [2]:
gamma_s = 26 #kN/m³
gamma = 16 #kN/m³
w = 0.03
n_max = 0.43
n_min = 0.36

Für die Verdichtung wurde gleichartiges Zugabematerial aus einer nahen Sandgrube gewonnen ($\gamma_\text{Grube}$, $w_\text{Grube}$). Bei Entnahme und Transport trat eine Auflockerung auf, sodass das aufgelockerte Material eine Wichte $\gamma_\text{Z} < \gamma_\text{Grube}$ aufwies. Die mittlere Zugabemenge dieses Materials betrug je Verdichtungspunkt $V_\text{Z}$.

In [3]:
gamma_Grube = 20 #kN/m³
w_Grube = 0.08
gamma_Z = 15 #kN/m³
V_Z = 2.6 #m³

Es sind nun folgende Fragen zu beantworten:

1) Welche (bezogenen) Lagerungsdichten $D$ und $I_D$ herrschten vor der Verdichtungsmaßnahme?

2) Wieviel m³ Sand mussten aus der Sandgrube entnommen werden?

3) Welche (bezogenen) Lagerungsdichten $D$ und $I_D$ wurde nach der Verdichtung erreicht?

4) Welche Wassermenge ist abzuführen, wenn das Grundwasser in konstanter Höhe bleiben soll?

## Lagerungsdichten vor der Verdichtungsmaßnahme

Die Lagerungsdichte $D$ und die bezogene Lagerungsdichte $I_D$ errechnen sich wie folgt:

$$
    D = \frac{n_\text{max} - n}{n_\text{max} - n_\text{min}} \qquad I_D = \frac{e_\text{max} - e}{e_\text{max} - e_\text{min}}
$$

Wir berechnen daher zunächst den Porenanteil 

$$
    e = \frac{V - V_\text{K}}{V_\text{K}} \quad \text{mit} \quad V_\text{K} = \frac{G_\text{d}}{\gamma_\text{s}} \quad \text{und} \quad V = \frac{G_\text{d}}{\gamma_\text{d}}
$$

Damit ergibt sich

$$
    e = \frac{\gamma_\text{s}}{\gamma_\text{d}} - 1
$$

Die Trockenwichte folgt aus

$$
    V = \frac{G_\text{d}}{\gamma_\text{d}} = \frac{G_\text{d} + G_\text{w}}{\gamma} \qquad \text{und somit} \qquad \gamma = \gamma_\text{d} \left(1 + \frac{G_\text{w}}{G_\text{d}} \right) = \gamma_\text{d} (1 + w)
$$

Aus dem Porenanteil folgt die Porenzahl (Porosität) und umekehrt:

$$
    n = \frac{e}{1+e} \qquad e = \frac{n}{1-n}
$$

In [4]:
gamma_d = gamma / (1 + w)

e = gamma_s / gamma_d - 1
n = e / (1 + e)

e_max = n_max / (1 - n_max)
e_min = n_min / (1 - n_min)

D = (n_max - n) / (n_max - n_min)
I_D = (e_max - e) / (e_max - e_min)

In [5]:
print("Die Trockenwichte beträgt %.2f kN/m³" %gamma_d)
print("Vor der Verdichtung ist die Porosität %.4f und die Porenzahl %.4f." %(n,e))
print("Maximale und minimale Porenzahl sind %.4f bzw. %.4f" %(e_max,e_min))
print("Damit beträgt die Lagerungsdichte %.1f Prozent ..." %(D*100))
print("... und die bezogene Lagerungsdichte %.1f Prozent" %(I_D*100))

Die Trockenwichte beträgt 15.53 kN/m³
Vor der Verdichtung ist die Porosität 0.4025 und die Porenzahl 0.6738.
Maximale und minimale Porenzahl sind 0.7544 bzw. 0.5625
Damit beträgt die Lagerungsdichte 39.2 Prozent ...
... und die bezogene Lagerungsdichte 42.0 Prozent


Das Ergebnis von $D = 0.392$ unterscheidet sich vom Ergebnis im Buch, in dem $D = 0.43$ angegeben ist. Dieser unterschied ist auf verfrühtes Runden zurückzuführen: Im Buch wird statt einer Porosität von $0.4025...$ auf $0.40$ gerundet:

In [6]:
(0.43 - 0.40)/(0.43-0.36)

0.4285714285714281

In [7]:
(0.43 - 0.402539)/(0.43-0.36)

0.39230000000000015

## Entnahmevolumen aus der Sandgrube

Das Zugabevolumen $V_\text{Z}$ je Verdichtungspunkt bezieht sich auf den aufgelockerten Zustand $\gamma_\text{Z}$. Wir rechnen nun das Volumen auf den Zustand $\gamma_\text{Grube}$ um. Dabei nehmen wir an, dass das erdfeuchte Material seinen Wassergehalt nicht ändert: $w_\text{Z} = w_\text{Grube}$. Da sich auch die Kornmasse an sich nicht ändert, bleiben Masse bzw. Gewicht des Bodens konstant (nur die Porosität ändert sich). Aus dieser Konstanz ergibt sich:

$$
    G = \gamma_\text{Grube} V_\text{Grube} = \gamma_\text{Z} V_\text{Z} \quad \text{und somit} \quad V_\text{Grube} = \frac{\gamma_\text{Z}}{\gamma_\text{Grube}} V_\text{Z}
$$

In [8]:
V_Grube = gamma_Z / gamma_Grube * V_Z
print("Je Verdichtungspunkt müssen der Sandgrube %.4f m³ Sand entnommen werden" %V_Grube)

Je Verdichtungspunkt müssen der Sandgrube 1.9500 m³ Sand entnommen werden


Die Anzahl $N$ der Verdichtungspunkte ergibt sich aus den Kantenlängen $a$, $b$ des verdichteten Bereichs und des Rechteckrasterabstandes $r$:

$$
    N = \frac{a}{r} \frac{b}{r}
$$

In [9]:
N = a/r * b/r
print("Die Anzahl der Verdichtungspunkte beträgt %i." %N)

Die Anzahl der Verdichtungspunkte beträgt 375.


In [10]:
V_Entnahme = V_Grube * N
print("Damit müssen der Grube für die Verdichtungsmaßnahme %.2f m³ Sand entnommen werden." %V_Entnahme)

Damit müssen der Grube für die Verdichtungsmaßnahme 731.25 m³ Sand entnommen werden.


## Lagerungsdichten nach der Verdichtung

Bei Annahme einer konstanten Geländehöhe beträgt das mittlere Korngewicht pro Rüttlerpunkt vor der Verdichtung:

$$
    G_\text{d} = V_\text{R} \gamma_\text{d} \qquad \text{mit} \quad V_\text{R} = r^2 z_\text{max}
$$

In [11]:
V_R = r**2 * z_max
G_d = V_R * gamma_d
V_KR = G_d/gamma_s
print("Gesamttrockengewicht vor Verdichtung beträgt %.2f kN je Verdichtungspunkt." %G_d)
print("Das entspricht einem Kornvolumen von %.2f m³." %V_KR)

Gesamttrockengewicht vor Verdichtung beträgt 621.36 kN je Verdichtungspunkt.
Das entspricht einem Kornvolumen von 23.90 m³.


Je Verdichtungspunkt wird nun Feststoff mit einem Gewicht von

$$
    G_\text{d,Z} = V_\text{Z} \gamma_\text{d,Z} = V_\text{Z} \frac{\gamma_\text{Z}}{1 + w_\text{Z}} = V_\text{Z} \frac{\gamma_\text{Z}}{1 + w_\text{Grube}}
$$

zugegeben.

In [12]:
G_dZ = V_Z * gamma_Z/(1 + w_Grube)
V_KZR = G_dZ/gamma_s
print("Trockengewicht des Zuschlagsstoffs: %.2f kN je Verdichtungspunkt." %G_dZ)
print("Das entspricht einem zusätzlichen Kornvolumen von %.2f m³." %V_KZR)

Trockengewicht des Zuschlagsstoffs: 36.11 kN je Verdichtungspunkt.
Das entspricht einem zusätzlichen Kornvolumen von 1.39 m³.


Bei Annahme eines gleichbleibenden Volumens ergibt sich die Trockenwichte nach Verdichtung je Verdichtungspunkt zu:

$$
    \gamma_\text{d,V} = \frac{G_\text{d} + G_\text{d,Z}}{V_\text{R}}
$$

In [13]:
gamma_dV = (G_d + G_dZ)/V_R
V_KV = V_KR + V_KZR
print("Die Trockenwichte des Bodens beträgt nach der Verdichtung %.2f kN/m³." %gamma_dV)
print("Das Kornvolumen des verdichteten Bodens je Verdichtungspunkt beträgt nun %.2f m³." %V_KV)

Die Trockenwichte des Bodens beträgt nach der Verdichtung 16.44 kN/m³.
Das Kornvolumen des verdichteten Bodens je Verdichtungspunkt beträgt nun 25.29 m³.


Wir können nun die Porenzahl und den Porenanteil nach der Verdichtung bestimmen. Es folgen sofort die Lagerungsdichtemaße.

In [14]:
e_V = gamma_s / gamma_dV - 1
n_V = e_V / (1 + e_V)

D_V = (n_max - n_V) / (n_max - n_min)
I_DV = (e_max - e_V) / (e_max - e_min)

In [15]:
print("Nach der Verdichtung ist die Porosität %.4f und die Porenzahl %.4f." %(n_V,e_V))
print("Damit beträgt die Lagerungsdichte %.1f Prozent ..." %(D_V*100))
print("... und die bezogene Lagerungsdichte %.1f Prozent" %(I_DV*100))

Nach der Verdichtung ist die Porosität 0.3678 und die Porenzahl 0.5818.
Damit beträgt die Lagerungsdichte 88.8 Prozent ...
... und die bezogene Lagerungsdichte 89.9 Prozent


## Abzuführende Wassermenge

Wir betrachten die Verhältnisse unterhalb des Grundwasserspiegels, da der Wassergehalt des eingebauten Materials oberhalb desselben $w_\text{max} = e \gamma_\text{w}/\gamma_\text{s}$ nicht übersteigt.

Vor der Verdichtungsmaßnahme beträgt das Wasservolumen im Gebiet:

$$
    V_\text{w0} = n V = n a b (z_\text{max} - z_\text{GW})
$$

Nach der Verdichtung ist bei gleichbleibendem Grundwasserflurabstand Raum für 

$$
    V_\text{wV} = n_V V = n_V a b (z_\text{max} - z_\text{GW})
$$

Die abzuführende Wassermenge ergibt sich aus der Differenz.

In [16]:
Delta_V_W = (n - n_V) * a * b * (z_max - z_GW)
print("Es sind %.1f m³ Wasser abzuführen." %Delta_V_W)

Es sind 416.7 m³ Wasser abzuführen.
